### PR TITLE
Template

### PR DESCRIPTION
**Pull request by tsohleDev**

### CODE REVIEW

**Comment by tsohleDev** [comment]
fixing style

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (Qoosim) [inline .gitignore:1]: - [ ] I would like to recommend that you should look into this file that is required to be ignored on your repo. I can still see the same file in your repo. Also, you're expected to add node_modules folder to the `.gitignore` file. Kindly follow this [link](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files) or [this](https://www.atlassian.com/git/tutorials/saving-changes/gitignore#:~:text=If%20you%20want%20to%20ignore,directory%20as%20an%20ignored%20file.) to take care of this.

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file will be ignore
```
Comment (Qoosim) [inline test.md:1]: - [ ] It is required that this file should be ignored inside your `.gitignore` file. Kindly fix it

**Review by Qoosim — COMMENTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @tsohleDev, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-   Good README file. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 
-   Fair descriptive PR title. :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] I can see that you set up the linters but I cannot see any of those files run on your PR. Kindly  look into why this is happening and fix it. Linters set up is very important in any of your project in Microverse. Any project with not linters setup would not be reviewed. Kindly fix this and ask for another review. Best of luck

File: test.md (hunk 2 above)
Comment (tsohleDev) [reply test.md:1]: Unfortunately I have mistakenly merged the the first pull request, and I cant fix the issues in this PR, but ive fixed the here
https://github.com/tsohleDev/Hello-Microverse/pull/2

**Pull request merged**